quick-start/build.gradle

```diff
@@ -42,6 +42,7 @@ dependencies {
         exclude group: 'com.google.guava', module: 'guava'
     }
     compile 'com.google.guava:guava:19.0'
+    compile 'org.springframework.boot:spring-boot-starter-aop:1.3.3.RELEASE'
```

Comment:
Hi @paxtonhare,

I compared the jar file created in master and in my branch, from 67.134229 MB, it became 69.004115 MB (so almost 2 MB). 

By the way, I noticed you merged my branch to your branch 'develop' and saw your changes. Would I still continue with the changes you requested?